e2e/authentication/signup.spec.js

```diff
@@ -1,30 +1,84 @@
-import { test } from '@playwright/test';
+import { test, expect } from '@playwright/test';
+// eslint-disable-next-line import/extensions
 import { resetDatabaseAndImportDataFromPath } from '../scripts/dts-import';
-import { ADMIN_EMAIL_ADDRESS, ADMIN_PASSWORD } from '../scripts/constants';
+import { fillValidSignUpForm } from './utils';
 
-test.describe('Authentication', () => {
+test.describe('Sign Up', () => {
   test.beforeEach(async ({ page }) => {
     await resetDatabaseAndImportDataFromPath({ filePath: './e2e/data/without-admin.tar' });
     await page.goto('/admin');
+    await fillValidSignUpForm({ page });
   });
 
-  test('a user should be able to signup when the strapi instance starts fresh', async ({
+  test('a user cannot submit the form if the first name field is not filled', async ({ page }) => {
+    expect(
+      await page.getByRole('textbox', { name: 'First name *' }).getAttribute('aria-required')
+    ).toBeTruthy();
```

Comment:
I wouldn't test it that way, because it tests internals: instead: I'd leave the field empty and submit the form, which shouldn't work.